Tool/src/build-in-docker.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+TARGETPLATFORM=${TARGETPLATFORM:-linux/amd64}
+BUILD_FOLDER=${TARGETPLATFORM/\//-}
+
+if [[ -d "/workspace/build/xray-${BUILD_FOLDER}" ]]; then
+  echo "Copying prebuilt binary"
+  cp /workspace/build/xray-${BUILD_FOLDER}/xray /workspace
```

Comment:
Looks like Dockerfile is picking binaries from `/workspace/xray` dir then shouldn't this be `cp /workspace/build/xray-${BUILD_FOLDER}/xray /workspace/xray` ? or am I missing something here?

Reply:
It is copying the file `xray` to the directory `/workspace`. I've added a trailing slash to make it more explicit